libsql/src/hrana/hyper.rs

```diff
@@ -37,23 +37,19 @@ impl HttpSender {
             .header(AUTHORIZATION, auth)
             .header("x-libsql-client-version", self.version.clone())
             .body(hyper::Body::from(body))
-            .unwrap();
+            .map_err(|err| HranaError::Http(format!("{:?}", err)))?;
```

Comment:
I'm using a `Debug` value here instead of a `Display` value. Its display value is `invalid format`, which is very difficult to troubleshoot. However, the debug value is `http::Error(InvalidUri(InvalidFormat))`, which is clear enough.